config/environments/production.rb

```diff
@@ -73,6 +91,16 @@
     end
   end
 
+  # Use a different logger for distributed setups.
+  # require "syslog/logger"
+  # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new "app-name")
+
+  if ENV["RAILS_LOG_TO_STDOUT"].present?
```

Comment:
I noticed this is a change from the previous behavior as well. Before this block was always run but now is behind an ENV var.

Reply:
Good catch ... I think arguably the env var approach is better, but for the purposes of this PR we just want to perform the upgrade and not change any other configuration, to the extent possible.

I'll update to keep the previous behavior.